### DISCUSSION TITLE
Disable (any) warning in lighting?

### DISCUSSION THREAD

**Discussion by innat** [Lightning-AI/pytorch-lightning#13328] [General]
Basic info
- torch version : 1.11.0
- lighting version : 1.6.3
- torchmetrics: 0.9.0

I want to disable the following warning message.

> /opt/conda/lib/python3.7/site-packages/torch/distributed/_sharded_tensor/__init__.py:10: DeprecationWarning: torch.distributed._sharded_tensor will be deprecated, use torch.distributed._shard.sharded_tensor instead
  DeprecationWarning
/opt/conda/lib/python3.7/site-packages/torchmetrics/utilities/prints.py:36: UserWarning: Torchmetrics v0.9 introduced a new argument class property called `full_state_update` that has
                not been set for this class (AdaptiveThreshold). The property determines if `update` by
                default needs access to the full metric state. If this is not the case, significant speedups can be
                achieved and we recommend setting this to `False`.
                We provide an checking function
                `from torchmetrics.utilities import check_forward_no_full_state`
                that can be used to check if the `full_state_update=True` (old and potential slower behaviour,
                default for now) or if `full_state_update=False` can be used safely.

**Comment by akihironitta** [comment]
Would you mind updating `torchmetrics` to `0.9.1` and trying it again? ref: https://github.com/Lightning-AI/metrics/pull/1058
